tests/e2e/virt_backup_restore_suite_test.go

```diff
@@ -22,15 +22,22 @@ var _ = Describe("VM backup and restore tests", Ordered, func() {
 		Expect(v).ToNot(BeNil())
 
 		if !v.IsVirtInstalled() {
-			err = v.EnsureVirtInstallation(5 * time.Minute)
+			err = v.EnsureVirtInstallation()
 			Expect(err).To(BeNil())
 			wasInstalledFromTest = true
 		}
+
+		err = v.EnsureEmulation(10 * time.Second)
+		Expect(err).To(BeNil())
+
+		err = v.EnsureDataVolume("openshift-cnv", "cirros-dv", "https://download.cirros-cloud.net/0.6.2/cirros-0.6.2-x86_64-disk.img", "128Mi", 5*time.Minute)
```

Comment:
Can we do something like?
```go
//"io/ioutil"
//"net/http"
//"strings"
//"fmt"
func getLatestCirrosImageURL() (string, error) {
    cirrosVersionURL := "https://download.cirros-cloud.net/version/released"
    
    resp, err := http.Get(cirrosVersionURL)
    if err != nil {
        return "", err
    }
    defer resp.Body.Close()
    
    body, err := ioutil.ReadAll(resp.Body)
    if err != nil {
        return "", err
    }
    
    latestCirrosVersion := strings.TrimSpace(string(body))
    
    imageURL := fmt.Sprintf("https://download.cirros-cloud.net/%s/cirros-%s-x86_64-disk.img", latestCirrosVersion, latestCirrosVersion)
    
    return imageURL, nil
}
```

Reply:
Sure, do you want to open a pull request to https://github.com/mrnold/oadp-operator/tree/ci-create-vm? Or should I just copy and paste this?